README_DFRAME_FORMAT.rst

```diff
@@ -33,9 +33,9 @@ Examples
 
 Structure example
 ^^^^^^^^^^^^^^^^^
-As shown below, an eframe of 4 chunks will be composed of a directory with each chunk file and the frame file::
+As shown below, an dframe of 4 chunks will be composed of a directory with each chunk file and the frame file::
```

Comment:
A dframe

Reply:
Fixed in 2426e5ae2a82ab94f3fc05902aeb22df78beb0b3 .